AutoGVP/add_INFO_fields.R

```diff
@@ -0,0 +1,59 @@
+################################################################################
+# add_INFO_fields.R
+# written by Ammar Naqvi
+#
+# This script annotates INFO column variables with variants that have been 
+# processed and re-called by AutoGVP
+#
+# usage: Rscript  add_INFO_fields.R  --input_table <tsv file> 
+#                                       --vcf_file <vcf file> 
+#                                       --out_prefix <string>
+################################################################################
+
+suppressPackageStartupMessages({
+  library("tidyverse")
+  library("optparse")
+  library("vroom")
+  library("vcfR")
+} )
+
+#Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+## set up directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+analysis_dir <- file.path(root_dir, "AutoGVP")
+results_dir <- file.path(analysis_dir, "results")
```

Comment:
```suggestion
results_dir <- file.path(root_dir, "results")
```
I had put this in root with a gitignore in #25 - we can discuss where it should really go, and where test data/results need to go, since they are a bit combined here (test data in input, while user data is also in input, this is confusing)